tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -926,4 +939,89 @@ public function renameGroup(string $oldGroup, string $newGroup, ?string $user =
 	public function userRenamesGroupUsingTheGraphApi(string $user, string $oldGroup, string $newGroup): void {
 		$this->featureContext->setResponse($this->renameGroup($oldGroup, $newGroup, $user));
 	}
+
+	/**
+	 * @param string $user
+	 *
+	 * @return ResponseInterface
+	 * @throws JsonException
+	 * @throws GuzzleException
+	 */
+	public function retrieveUserInformationUsingGraphApi(
+		string $user
+	):ResponseInterface {
+		$credentials = $this->getAdminOrUserCredentials($user);
+		return GraphHelper::getOwnInformationWithGroups(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$credentials["username"],
+			$credentials["password"],
+		);
+	}
+
+	/**
+	 * @When /^the user "([^"]*)" retrieves (her|his) information using the Graph API$/
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 * @throws JsonException
+	 */
+	public function userRetrievesHisorHerInformationOfUserUsingGraphApi(
+		string $user
+	):void {
+		$response = $this->retrieveUserInformationUsingGraphApi($user);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * @Then /^the API response should contain the following information:$/
```

Comment:
The step also seems too generic. The implementation is not targeted to check any API response I think.
```suggestion
	 * @Then /^the user retrieve API response should contain the following information:$/
```

Reply:
`* @Then /^the user retrieved API response should contain the following information`
or could be
`* @Then /^the last response should contain the following user information`